pkg/event/events.go

```diff
@@ -125,3 +125,37 @@ func NewBackgroundSuccessEvent(policy, rule string, source Source, r *unstructur
 
 	return events
 }
+
+func NewPolicyExceptionEvent(engineResponse *response.EngineResponse, ruleResp *response.RuleResponse) Info {
+	var reasonBuilder, messageBuilder strings.Builder
+	defer reasonBuilder.Reset()
+	defer messageBuilder.Reset()
+
+	exceptionName, exceptionNamespace := getExceptionEventInfoFromRuleResponseMsg(ruleResp.Message)
+
+	fmt.Fprintf(&reasonBuilder, "exception %s/%s matches resource %s", exceptionNamespace, exceptionName, engineResponse.PatchedResource.GetName())
```

Comment:
The reason should be PolicySkipped as defined here:

[kyverno/pkg/event/reason.go](https://github.com/kyverno/kyverno/blob/03cec01fb5efc0f801eb5203103fa163a0233a81/pkg/event/reason.go#L7)

Reply:
Make sense!